impl/client/src/main/javascript/web/dataapi/nls/messages_de.properties

```diff
@@ -15,6 +15,7 @@ datasourcelistlbl_content=Datenquellen:
 editdatasourceimg_title=Die ausgew\u00e4hlte Datenquelle bearbeiten
 # deletedatasourceimg_title is the tooltip for the 'delete datasource' button on the datasource selection dialog
 deletedatasourceimg_title=Die ausgew\u00e4hlte Datenquelle l\u00f6schen
+DeleteDatasourceWarningTitle=
```

Comment:
I doubt that you want an empty string as your translation. Those empty properties should be removed so that the translation can be looked up in the parent bundle.

(And yeah, be prepared to fix a few of these problems when weaving in newly translated strings. Its even more fun once you get to decide whether to encode or not to encode strings with entities based on where and how the string is used.)